README.md

```diff
@@ -60,6 +61,11 @@ A curated list of awesome GraphQL Security frameworks, libraries, software and r
 - [GraphQL Visualizer](https://www.graphqlvisualizer.com) - Visualize GraphQL schema.
 - [Voyager](https://github.com/IvanGoncharov/graphql-voyager) - Represent any GraphQL API as an interactive graph.
 - [GraphQL Inspector](https://github.com/kamilkisiela/graphql-inspector) – Validate schema, get schema change notifications, validate operations, find breaking changes, look for similar types, schema coverage.
+- [GraphQL Rover](https://github.com/Brbb/graphql-rover) - GraphQL schema viewer (requires URL + introspection, can't paste JSON)
+- [Nova](https://github.com/nova-introspection/Nova) / [NovaQL](https://github.com/nova-introspection/novaql) - GraphQL schema viewer (requires URL + introspection, can't paste JSON)
```

Comment:
Theses two entries are not formatted accordingly to awesome-list format. 1 entry per line and no explanation parenthesis.

Also `Nova` is no longer maintained.

Reply:
I fixed the formatting.